subsystem/leader_test.go

```diff
@@ -111,6 +111,11 @@ func getHomeDir() string {
 }
 
 var _ = Describe("Leader tests", func() {
+	if Options.DeployTarget != "k8s" {
+		log.Info("Leader tests are disabled for non-k8s deployments")
```

Comment:
Use `Skip`

```golang
if Options.DeployTarget != "k8s" {
    Skip("Leader tests are disabled for non-k8s deployments")
}
```

Reply:
I had tried that. Skip() only works at test level, within an It or BeforeEach.